pandas/tools/util.py

```diff
@@ -173,6 +174,8 @@ def to_numeric(arg, errors='raise', downcast=None):
             values = arg.values
     elif isinstance(arg, (list, tuple)):
         values = np.array(arg, dtype='O')
+    elif isinstance(arg, Decimal):
```

Comment:
ok this is what I'd like to see; we are trying to treat ``decimal.Decimal`` in a more thoughtful way

```
In [7]: d = decimal.Decimal('1.0')

In [8]: pandas.types.common.is_scalar(d)
Out[8]: False

In [9]: pandas.types.common.is_number(d)
Out[9]: True
```

I'd like [7] to be True.
further I'd like to add a ``is_decimal`` in ``pandas.types.inference``

then the last part can be something like:
```
    elif isinstance(arg, (list, tuple)):
        values = np.array(arg, dtype='O')
    elif is_scalar(arg): # previous np.isscalar(arg):
        if is_decimal(arg):
            # coerce
            arg = float(arg)
        if is_number(arg):
            return arg
        is_scalar = True
        values = np.array([arg], dtype='O')
```